backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -162,7 +162,9 @@ private List filterUsersByRole(Optional<String[]> selectedRoles, String clientId
             roles = selectedRoles.get();
         }
         for (String role:roles){
-            ResponseEntity res = webClientService.getUsersInRole(clientId,role,null);
+            MultiValueMap<String, String> queryParam = new LinkedMultiValueMap<>();
```

Comment:
I'd suggest calling this map variable `queryParams` even though you're currently only setting one parameter since it is a map that could hold more than one parameter (say, if we ever needed to add anything in the future).